addons/mail/static/src/components/chatter_topbar/chatter_topbar.xml

```diff
@@ -2,15 +2,17 @@
 <templates xml:space="preserve">
 
     <t t-name="mail.ChatterTopbar" owl="1">
-        <div class="o_ChatterTopbar" t-attf-class="{{ className }}" t-ref="root">
+        <div class="o_ChatterTopbar justify-content-between d-flex" t-attf-class="{{ className }}" t-ref="root">
             <t t-if="chatter">
-                <div class="o_ChatterTopbar_actions" t-att-class="{'o-has-active-button': chatter.composerView }">
+                <!-- Use flex-wrap-reverse to ensure send buttons are directly above composer -->
+                <div class="o_ChatterTopbar_actions flex-wrap-reverse flex-fill d-flex border-bottom" t-attf-class="{{ chatter.composerView ? 'o-has-active-button' : 'border-white' }}">
```

Comment:
Hey @alexkuhn 👋 
This is necessary when the top bar must be divided into several lines.

![topbar](https://user-images.githubusercontent.com/80679690/158794839-86f5e35c-113e-4a5e-af4c-8526eaa6fada.jpg)

Reply:
Ok, both UI are quite broken, but the `flex-wrap-reverse` version is indeed better.
I would remove the comment, because this wastes a line in XML and it'd be unfair that only `flex-wrap-reverse` has special treatment with having its own comment.

Naturally, let's not change UI. But this is small fix so it's fine in this PR. Thanks :)

Reply:
It's done 👍